apps/api/package.json

```diff
@@ -39,46 +39,12 @@
     "@nestjs/platform-express": "9.3.12",
     "@nestjs/swagger": "6.0.1",
     "@nestjs/terminus": "9.2.1",
-    "@novu/apns": "^0.13.0",
     "@novu/application-generic": "^0.13.0",
-    "@novu/burst-sms": "^0.13.0",
-    "@novu/clickatell": "^0.13.0",
     "@novu/dal": "^0.13.0",
-    "@novu/discord": "^0.13.0",
-    "@novu/emailjs": "^0.13.0",
-    "@novu/expo": "^0.13.0",
-    "@novu/fcm": "^0.13.0",
-    "@novu/firetext": "^0.13.0",
-    "@novu/forty-six-elks": "^0.13.0",
-    "@novu/gupshup": "^0.13.0",
-    "@novu/infobip": "^0.13.0",
-    "@novu/kannel": "^0.13.0",
-    "@novu/mailersend": "^0.13.0",
-    "@novu/mailgun": "^0.13.0",
-    "@novu/mailjet": "^0.13.0",
-    "@novu/mandrill": "^0.13.0",
-    "@novu/mattermost": "^0.13.0",
-    "@novu/ms-teams": "^0.13.0",
-    "@novu/netcore": "^0.13.0",
```

Comment:
before we have had two places in the code where we used providers/handlers/factories - API and application-generic package, now this code in only in the `application-generic` and is reused between API and worker-service app